src/Lykke.Service.PayInternal.Contract/PaymentRequest/PaymentRequestPublicStatuses.cs

```diff
@@ -44,5 +44,20 @@ public static class PaymentRequestPublicStatuses
         /// Refund error
         /// </summary>
         public const string RefundError = "REFUND_ERROR";
+
+        /// <summary>
+        /// Payment request is accepted to settlement.
+        /// </summary>
+        public const string SettlementInProgress = "SETTLEMENT_INPROGRESS";
```

Comment:
These new public statuses must be mapped to public statuses in public API: https://github.com/LykkeCity/LykkePayAPI/blob/dev/src/Lykke.Service.PayAPI/ConvertExtensions.cs#L60

Reply:
LPDEV-902 is created